docs/book/src/declare/06_mix_kinds.md

```diff
@@ -1,7 +1,7 @@
 # Mixing Kinds
 
-What happen if for a key you declare plurals in one locale, interpolation in another, and a simple string in a third ?
+What happens if for a key you declare plurals in one locale, interpolation in another, and a simple string in a third ?
 
-Well this is totally allowed, but you will still need to supply all values/component of every locales combined when using the translation, regardless of what the current locale is.
+Well this is totally allowed, but you will still need to supply all values/components of every locale combined when using the translation, regardless of what the current locale is.
 
-What is not allowed to mix is subkeys, the key must be subkeys in all locale.
+What is not allowed to mix are subkeys, the key must be a subkey in all locales.
```

Comment:
This is a difficult thing to phrase, my intent was to explain that if a key contain subkeys, that key must contain subkeys in all other locales, `"the key must be a subkey"` feels like the key being one depth deeper.

Reply:
How about we change it to something like this:
`"What is not allowed to mix are subkeys. If a key has subkeys in one locale, the key must have subkeys in all locales."`

Reply:
Perfect!